scripts/install_bootstrap

```diff
@@ -1,6 +1,10 @@
 #! /bin/bash
 set -e
 PANCANCER_LAUNCHER_VERSION=3.1.7
+PANCANCER_CLI_VERSION=0.0.7
+WORKFLOW_LISTING_URL=https://github.com/ICGC-TCGA-PanCancer/cli/blob/${PANCANCER_CLI_VERSION}/config/workflowlist.json
```

Comment:
This will tie the repo to a specific release version, which is already a problem with the current system of hard-coding the URLs into workflowlister.py. 
Should we begin moving the workflow list file to somewhere else to avoid this? Other places could be this repository but always refer to develop (or maybe a specific feature branch), or a completely different repository.